addons/api/tests/unit/serializers/credential-test.js

```diff
@@ -97,7 +97,7 @@ module('Unit | Serializer | credential', function (hooks) {
     const store = this.owner.lookup('service:store');
     const serializer = store.serializerFor('credential');
     const record = store.createRecord('credential', {
-      private_key_passphrase: '',
+      private_key_passphrase: null,
```

Comment:
What's the reason for this change?

Reply:
That was just to verify that if the value was `null` the `isSecret` logic would still work as expected in the test. If the user somehow set the value to null, we still don't want that sent to the API.